SohamGaikwad/Advanced/NewsApp/app/src/main/java/com/example/newsapp/DetailsActivity.java

```diff
@@ -0,0 +1,43 @@
+package com.example.newsapp;
+
+import android.content.Intent;
+import android.graphics.Color;
+import android.graphics.drawable.ColorDrawable;
+import android.os.Bundle;
+import android.view.MenuItem;
+import android.webkit.WebView;
+
+import androidx.appcompat.app.ActionBar;
+import androidx.appcompat.app.AppCompatActivity;
+
+public class DetailsActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_details);
+
+        WebView editWeb = findViewById(R.id.web);
+        Intent intent = getIntent();
+
+        String url = intent.getStringExtra("url");
+        editWeb.loadUrl(url);
+
+        ActionBar bar = getSupportActionBar();
+        bar.setTitle("News");
```

Comment:
Please handle NullPointerException here and other similar places.

Reply:
Fixed in ad34fcde338dffceb4d0ffa797d1c1bfc738641e.